plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/GoogleLoginService.java

```diff
@@ -28,17 +42,59 @@
  */
 public class GoogleLoginService {
 
-  // TODO(chanseok): this will be needed later.
-  public static final List<String> OAUTH_SCOPES = Collections.unmodifiableList(Arrays.asList(
-      "https://www.googleapis.com/auth/userinfo#email"
+  private static final List<String> OAUTH_SCOPES = Collections.unmodifiableList(Arrays.asList(
```

Comment:
Not sure our plan is to allow incremental login, i.e., starting with virtually no scope and then incrementally adding new scopes by presenting a consent screen whenever a user requires additional scope. That sounds complex, so I think at least for now we will allow every scope we are going to use in our tools, including scopes for, e.g., Stackdriver Debugger, Source Repository, App Engine, etc. So, it's not just for App Engine.